src/keypair.rs

```diff
@@ -93,6 +100,24 @@ impl FromStr for Keypair {
                     })?;
                 Ok(keypair.into())
             }
+            #[cfg(feature = "tpm")]
+            Some("tpm") => {
+                let args = KeypairArgs::from_uri(&url).map_err(error::DecodeError::keypair_uri)?;
+                let network = args.get("network", Network::MainNet)?;
+                let path = url.path();
+
+                let keypair = tpm::init()
+                    .map_err(|err| uri_error!("could not initialize tpm \"{path}\": {err:?}"))
+                    .and_then(|_| {
+                        tpm::Keypair::from_key_path(network, path)
+                            .map(helium_crypto::Keypair::from)
+                            .map_err(|err| {
+                                uri_error!("could not load tpm keypair on path {path}: {err:?}")
+                            })
+                    })?;
```

Comment:
since tpm::init() is hidden now:
```suggestion
                let keypair = tpm::Keypair::from_key_path(network, path)
                    .map(helium_crypto::Keypair::from)
                    .map_err(|err| {
                        uri_error!("could not load tpm keypair on path {path}: {err:?}")
                    })?;
```